nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_DisconnectLine/reference/outputs/lostEquipments/lostEquipments.xml

```diff
@@ -1,2 +1,4 @@
 <?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
-<lostEquipments xmlns="http://www.rte-france.com/dynawo"/>
+<lostEquipments xmlns="http://www.rte-france.com/dynawo">
+  <lostEquipment id="_BUS____1-BUS____5-1_AC" type="LINE"/>
```

Comment:
Why is that? Since the modification is more strict (only purely disconnected nodes) I would have expected less items, not more?

Reply:
Me too. It's the revert of the same commit when the PR switched from connected to PartiallyConnected. There are some things to investigate with this change that was not done well and need to solve it in another PR